cypress/integration/createSignalWrak.spec.js

```diff
@@ -0,0 +1,144 @@
+// <reference types="Cypress" />
+import * as createSignal from '../support/commandsCreateSignal';
+import { SIGNAL_DETAILS } from '../support/selectorsSignalDetails';
+
+describe('Create signal parkeeroverlast and check signal details', () => {
+  describe('Create signal parkeeroverlast', () => {
+    before(() => {
+      cy.visitFetch('incident/beschrijf');
+    });
+
+    it('Should describe the signal', () => {
+      cy.server();
+      cy.defineGeoSearchRoutes();
+      cy.getAddressRoute();
+      cy.route('POST', '**/signals/category/prediction', 'fixture:fietsWrak.json').as('prediction');
+
+      createSignal.checkDescriptionPage();
+      createSignal.setAddress('1066CD 1045', 'Sloterweg 1045, 1066CD Amsterdam');
+      createSignal.setDescription(
+        'Voor het Velodrome ligt een fietswrak, het lijkt een beetje op die van Tom Dumoulin.'
+      );
+      createSignal.setDateTime('Nu');
+
+      cy.contains('Volgende').click();
+    });
+
+    it('Should enter specific information', () => {
+      createSignal.checkSpecificInformationPage();
+
+      cy.contains('Voor het Velodrome ligt een fietswrak, het lijkt een beetje op die van Tom Dumoulin.').should(
```

Comment:
Is it on purpose you use the same string, or  would it be nicer to use a variable? I see the `Voor het Velodtrome` string a couple of times.